torch/_dynamo/variables/torch_function.py

```diff
@@ -73,10 +73,11 @@ def _is_attr_overidden(tx, var, name):
 def call_torch_function(
     tx, torch_function_type, torch_function_var, fn, types, args, kwargs
 ):
+    from .builder import SourcelessBuilder
     # signature:
     # def __torch_function__(cls, func, types, args=(), kwargs=None):
-    tf_args = (torch_function_type, fn, types, TupleVariable(list(args)))
-    return tx.inline_user_function_return(torch_function_var, tf_args, kwargs)
+    tf_args = (torch_function_type, fn, types, SourcelessBuilder()(tx, tuple(args)), SourcelessBuilder()(tx, kwargs))
+    return tx.inline_user_function_return(torch_function_var, tf_args, {})
```

Comment:
dumb q not know enough about dynamo but does this work with kwarg only args

Reply:
Yeah it will,  basically, all kwargs will just get put into the kwargs dict which is the last argument. The key to thinking about this is that the signature of torch function is independent of the actual op.